charts/fleet/Chart.yaml

```diff
@@ -4,8 +4,9 @@ name: fleet
 keywords:
   - fleet
   - osquery
-version: v4.17.0
+version: v4.18.0
 home: https://github.com/fleetdm/fleet
 sources:
   - https://github.com/fleetdm/fleet.git
-appVersion: v4.17.0
+appVersion: v
```

Comment:
Heads up this typo would break Helm deployments. I'm fixing it while merging `main`.

Reply:
@zwass Thanks so much! I'm not sure how that happened. Will add to my list to triple check!